src/kubernetes.rs

```diff
@@ -47,7 +83,11 @@ impl Kubernetes {
         let vote_key_path = config_dir.join("bootstrap-validator/vote-account.json");
         let stake_key_path = config_dir.join("bootstrap-validator/stake-account.json");
 
-        let mut secrets = HashMap::new();
+        let bootstrap_keypair = read_keypair_file(&identity_key_path)
+            .expect("Failed to read bootstrap validator keypair file");
+        self.add_known_validator(bootstrap_keypair.pubkey());
```

Comment:
nit: just out of curiosity, do we add this one for the rest of validator scripts 🤔 also, it seems that this action is not quite related to the `create_bootstrap_secret`

Reply:
you are def right that this is in the wrong spot. not related to `create_bootstrap_secret`. will fix. I added the bootstrap to `known_validators` because for all other validators deployed, I will pass in `--known-validators <bootstrap-identity>` to the other validators' startup scripts